tests/unittests/Caffe2ImporterTest.cpp

```diff
@@ -67,6 +67,43 @@ TEST(caffe2, importConv) {
     EXPECT_FLOAT_EQ(result.raw(i), expectedValues[i]);
 }
 
+TEST(caffe2, importConvRelu) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename(
+      GLOW_DATA_PATH "tests/models/caffe2Models/convrelu_pred_net.pbtxt");
+  std::string NetWeightFilename(
+      GLOW_DATA_PATH "tests/models/caffe2Models/convrelu_init_net.pbtxt");
+
+  Placeholder *output;
+  PlaceholderBindings bindings;
+
+  {
+    Tensor data;
+    getNCHWData(&data, 1, 1, 3, 3);
+    Caffe2ModelLoader caffe2LD(NetDescFilename, NetWeightFilename,
+                               {"gpu_0/data_0"}, {&data.getType()}, *F);
+    output = EXIT_ON_ERR(caffe2LD.getSingleOutput());
+
+    bindings.allocate(mod.getPlaceholders());
+    updateInputPlaceholdersByName(bindings, &mod, {"gpu_0/data_0"}, {&data});
+  }
+
+  auto res = bindings.get(output);
+  EE.compile(CompilationMode::Infer, F);
```

Comment:
for importer tests the goal is to verify structure, can you check that there are Conv, Relu nodes here.